phys2bids/tests/test_txt.py

```diff
@@ -160,3 +160,15 @@ def test_raises():
     with raises(AttributeError) as errorinfo:
         txt.process_labchart(channels, chtrig, header)
     assert 'not in a valid LabChart' in str(errorinfo.value)
+
+
+def test_muiltifreq():
+    # testing file already downloaded in the first test
```

Comment:
I know the files are downloaded once already, but due to what happened during #145, I would rather download and delete the files for every test function - just to avoid problems.

Reply:
That will significantly increase our testing time, but we can do it

Reply:
done